modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -63,7 +63,11 @@ ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1
 rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0 cio_ignore=all,
 !condev rd.dasd=0.0.3490
 ----
-
++
+[NOTE]
+====
+All options in parmfile should be written as a single line, please make sure you have no newlines characters.
```

Comment:
@nikita-dubrovskii I would change the note to: _Write all options in parmfile as a single line and make sure you have no newline characters._ 

And you have to ask the openshift docs team for peer review.